bio/mmseqs2/workflows/wrapper.py

```
@@ -0,0 +1,37 @@
+__license__ = "MIT"
+
+import os
+import tempfile
+from snakemake.shell import shell
+from snakemake_wrapper_utils.snakemake import get_format
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+
+target = snakemake.input.get("target", "")
```

Ruff (0.13.1)

Undefined name `snakemake`

(F821)